projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -6757,7 +6769,7 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
         if (this.hasColumnsToAutosize) {
             this.cdr.detectChanges();
             this.zone.onStable.pipe(first()).subscribe(() => {
-                this.autoSizeColumnsInView();
+                this._autoSizeColumnsNotify.next();
```

Comment:
Just `this.zone.onStable.pipe(first()).subscribe(this._autoSizeColumnsNotify);` should do too

Reply:
It doesn't seem to trigger correctly like that:
![image](https://github.com/IgniteUI/igniteui-angular/assets/10397980/fe54ae4e-543b-4ac8-a8d5-343a905de8a6)